START-HERE.md

```
@@ -0,0 +1,99 @@
+
+---
+
+## 3. Immediate Action: The Smallest Viable Task
+
+### 3.1. Human Pathway (3-Step Quickstart)
+
+To establish a verified local workspace, execute the following command chain:
+
+1. **Environment Initialization:**
+   Ensure Python 3.12+ and `uv` are installed. Synchronise the virtual environment:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)